tools/pd-ctl/pdctl/ctl_test.go

```diff
@@ -0,0 +1,73 @@
+package pdctl
```

Comment:
Please add the license

Reply:
what license?

Reply:
// Copyright 2016 TiKV Project Authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// See the License for the specific language governing permissions and
// limitations under the License.

?